application/src/main/java/org/opentripplanner/updater/spi/PollingGraphUpdater.java

```diff
@@ -40,7 +41,15 @@ public abstract class PollingGraphUpdater implements GraphUpdater {
   /**
    * Parent update manager. Is used to execute graph writer runnables.
    */
-  protected WriteToGraphCallback saveResultOnGraph;
+  private WriteToGraphCallback saveResultOnGraph;
```

Comment:
This is not a new change, but it still seems a bit odd to me that the interface and field have different names here, with the field name being a verb. There's also a comment below referring to this as the "GraphWriter" and I'm not sure anything still has that name. Probably all a result of a series of renamings as the callback interface was extracted from the update manager.